source/User_Guide/email_activity_beta.md

```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
 
 With the Email Activity feed you can:
 
-* Troubleshoot email delivery issues faster with comprehensive and sequential event data.
-* Pinpoint specific emails easily with advanced search and filter options -- including subject line and email metadata.
+* Troubleshoot email delivery issues faster with comprehensive and sequential event data by email.
+* Pinpoint specific emails easily with advanced search and filter options --including subject line and email metadata.
 * Customize your data independently with CSV download/export option.
 * Increase storage available for up to 60 days for historical visibility within the UI by purchasing an upgrade.
-* Complete an API call to gather specific email events when additional storage is purchased.
+* Call all events by an email from the API with additional storage purchase.
 
 {% info %}
-SendGrid stores two days of Email Activity data for free. To increase your storage capability, see LINK
+SendGrid stores two days of Email Activity data for free. To increase your storage capability, you may purchase more storage [here](https://app.sendgrid.com/settings/billing).
 {% endinfo %}
 
-{% anchor h2 %}
+{% anchor H2 %}
+Types of email activity data
+{% endanchor %}
+
+The Email Activity feed allows you to view specific information about messages sent from your account. 
+
+
+<table class="table" style="table-layout:fixed">
+  <tr>
+  <th class="tg-031e">Event</th>
```

Comment:
also, I just realized that you should delete all of these random classes. That's generated automatically by the table generator website, but you can turn it off. Click the box for "Do not generate CSS".